fsmonitor-ipc.h

```diff
@@ -1,6 +1,8 @@
 #ifndef FSMONITOR_IPC_H
 #define FSMONITOR_IPC_H
 
+#include "simple-ipc.h"
```

Comment:
Yes, I squashed this into 2/5 in part 2.

Reply:
Correct, but forgot the Helped-by, which might be why I am still waiting on that github recruiter call ;)